### ISSUE TITLE
add hover styles to Transcription Group

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#177]
The `<input>` fields of a Transcription Group should get a blue highlight when focused, just like text fields. Their associated help text and placeholder content should also change color to blue.

The `:focus-within` selector will probably be useful for getting help text to change color whenever the user's focus is inside the `<fieldset>`.

**Comment by vadekh** [comment]
@dwhieb The `:focus-within` selector works, but since we use `<fieldset>` tags in larger scopes (i.e. the additional names field) applying the color change to all the help text inside a `fieldset` will not have the desired effect.
<img width="635" alt="Screen Shot 2021-09-30 at 12 31 34 PM" src="https://user-images.githubusercontent.com/36901345/135503158-375e80de-74a2-496b-a25d-22cff14d5510.png">

**Comment by dwhieb** [comment]
(Quick note: You should test this for the Language Name or Autonym fields. The Additional Name field doesn't have a Transcription Group in it. Since Transcription Groups only ever have 1 help text element, I don't think you'll run into the problem you mentioned above.)

I think the declarations you need might look something like this:

```less
fieldset:focus-within {
  .help-text {
    color: var(--blue);
  }
}

input:focus {
  /* blue border styles */
}
```

Since some of the styles here are going to be the same as ones in the Input Group class, I think we should create a `--focus-border` variable in `borders.less`, and use that variable in both the Input Group and Transcription Group.

**Comment by vadekh** [comment]
I am unclear on how to select the parent fieldset of a Transcription Group, without making changes to the `fieldset.less` file. Wouldn't the above changes have no effect on any fieldsets because the less file only styles elements within the `.txn-group` class?
Then, making broader changes to the `fieldset.less` file results in the screenshot above.

**Comment by dwhieb** [comment]
Yup, you're totally right—my mistake!

**tl;dr for below:** Let's abandon the requirement of making the help text blue on hover.

I see a couple options then:

1. Ditch the requirement about making the help text blue (I was on the fence about that anyway), and just make the text input elements themselves have the blue border and placeholder text.

2. Make the help text blue by adding styles the Language Editor for the individual fieldsets. I don't think that'd be a very maintainable way to implement this, because we'd have to add styles for every single fieldset containing a transcription group in every single form on every page of the app.

3. Add a `.focusable` variant to the `.fieldset` class like this:

    ```less
    .fieldset {

      /* other styles */

      .focusable {
        .help-text:hover {
          color: var(--blue);
        }
      }
  
    }
    ```

    Then we'd have to add the `.focusable` class to any `.fieldset` that contains just a Transcription Group.

Honestly, I'm fine with either (1) or (3). (3) is a nice solution, but does create additional maintenance for a feature that isn't really crucial. We'd have to make sure the relevant fieldsets always had the `.focusable` class. So I'm leaning towards (1)—just abandon styling the help text.

**Comment by vadekh** [comment]
@dwhieb (1) sounds good to me. That was my initial thought as well since the help text placement is a bit different for Transcription Group elements. The text input highlighting has pretty much the same effect. Ill make a PR for this issue tomorrow.
